testing/.env.example

```diff
@@ -30,12 +30,6 @@ BROWSERSTACK_DEBUG_MODE=false
 # i.e. iPad7th_13 --> iPad 7th generation, Running Browser Version(iOS), 13
 BROWSERSTACK_CONFIGURATION_OPTIONS=Windows_10_83
 
-# Whether you want to run on the Selenium Grid or not?
-GRID=false
```

Comment:
I'm a bit against removing these from the example file, as these env params will still work and can be used locally by someone spinning up a grid.

We will also definitely be doing some more work on this once 4.0.0 is out as I'm hearing then the edge image for docker will be a bit better (Possibly).

Reply:
Oh interesting, I hadn't considered someone using grid locally. We don't have any need for it in github actions land anymore. So perhaps the best thing to do is instead to update the examples to they are more generic; not referring to docker-compose endpoints that no longer exist.

Reply:
Tell you what. To avoid holding this up. Lets delete these for now. And "if" we want to use them again in DS, we'll just re-add them in.

Reply:
Hah! I went the other way 😄.